app/src/main/java/com/github/polypoly/app/menu/ProfileActivity.kt

```diff
@@ -168,7 +184,7 @@ class ProfileActivity : ComponentActivity() {
         Box(
             modifier = Modifier
                 .clip(CircleShape)
-                .background(color = if (won) MaterialTheme.colors.primary else Color.DarkGray)
+                .background(color = if (won) MaterialTheme.colors.secondary else purpleVeryDark)
```

Comment:
Here too, even if you use the colors defined in `PolypolyTheme`, I think it's better if we really only stick to the palettes and only the palettes.